plugin/evm/vm.go

```diff
@@ -380,7 +453,12 @@ func (vm *VM) Initialize(
 	default:
 		lastAcceptedHash = common.BytesToHash(lastAcceptedBytes)
 	}
-	ethChain, err := coreth.NewETHChain(&ethConfig, &nodecfg, vm.chaindb, vm.config.EthBackendSettings(), vm.createConsensusCallbacks(), lastAcceptedHash, &vm.clock)
+
+	var err error
+
+	vm.atomicTransactor = newAtomicTransactor(vm.ctx.AVAXAssetID, vm.ParseAddress, vm.newExportTx)
```

Comment:
Initialization of the `AtomicTransactor`. We grab the AVAX asset ID here so we can use it later when we receive the string literal `"AVAX"`

Reply:
Additionally, we grab a reference to two methods that we need to call.